peewee_async/databases.py

```diff
@@ -39,6 +44,9 @@ def is_connected(self) -> bool:
     async def aio_close(self) -> None:
         """Terminate pool backend. The pool is closed until you run aio_connect manually
         """
+        if self.deferred:
+            raise Exception('Cannot connect before db inited')
```

Comment:
Let's use this text: **Error, database must be initialized before creating a connection pool.**